istioctl/cmd/istioctl/proxyconfig.go

```diff
@@ -60,8 +59,7 @@ var (
 		Aliases: []string{"clusters", "c"},
 		Args:    cobra.ExactArgs(1),
 		RunE: func(c *cobra.Command, args []string) error {
-			podName := args[0]
-			ns := handleNamespace()
+			podName, ns := getProxyDetails(args[0], namespace)
```

Comment:
this should still be using handleNamespace, no?
```suggestion
			podName, ns := inferPodInfo(args[0], handleNamespace())
```

Reply:
yes, have made those changes.